### DISCUSSION TITLE
How to change colors of titles in favorites?

### DISCUSSION THREAD

**Discussion by redfusion83** [punxaphil/custom-sonos-card#462] [Q&A]
Any tipps on how to change the background and font color of the titles while browsing favorites with mod-card?

![Screenshot_20240916_081121_Home Assistant](https://github.com/user-attachments/assets/0e01eb6c-50db-455e-a8be-5b3092fd5f73)

**Comment by punxaphil** [comment]
Using https://github.com/thomasloven/lovelace-card-mod:
```
type: custom:sonos-card
card_mod:
  style:
    sonos-media-browser$ sonos-media-browser-icons$: |
      .title {            
        color: red !important;
      }  
```
---
PunxaPhil
[![image](https://github.com/johanfrick/custom-sonos-card/assets/26582898/9bebc1f6-6bda-4cbd-b596-50d3d2f712de)](https://www.buymeacoffee.com/punxaphil)
